sdk/appcenter-crashes/src/test/java/com/microsoft/appcenter/crashes/CrashesTest.java

```diff
@@ -1565,8 +1528,10 @@ public void minidumpStoredWithOldSDK() throws Exception {
         mockStatic(ErrorLogHelper.class);
         mockStatic(ErrorAttachmentLog.class);
         ErrorReport errorReport = new ErrorReport();
-        errorReport.setThrowable(new NativeException());
-        when(ErrorLogHelper.getErrorReportFromErrorLog(any(ManagedErrorLog.class), any(Throwable.class))).thenReturn(errorReport);
+        Device device = new Device();
+        device.setWrapperSdkName(WRAPPER_SDK_NAME_NDK);
+        errorReport.setDevice(device);
```

Comment:
Is this added because of validation failure? Just a question.

Reply:
We were testing the type of the deserialized throwable type which does not exist anymore. Now it's testing the device wrapper sdk name instead (to know its breakpad).